### ISSUE TITLE
minor typo in assert_attributions_order function

### ISSUE THREAD

**Issue by jang1suh** [understandable-machine-intelligence-lab/Quantus#273]
### Description
Thanks for the great project!
I have tried to work with `region_perturbation` metric, but this function does not accept order="lerf".
I think there is a typo in the function for assertion:
https://github.com/understandable-machine-intelligence-lab/Quantus/blob/f94853b653700b8f499d203118664edf8ad8bef3/quantus/helpers/asserts.py#L109-L126
`region_perturbation` metric only accepts either "lerf" or "morf" for order, so "lerf" instead of "lorf" should be used in the above function.

### Steps to reproduce the behavior
```
import torch
import numpy as np
from quantus.metrics.faithfulness.region_perturbation import RegionPerturbation

class TestModel(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.classifier = torch.nn.Linear(2,1)
    def forward(self, x):
        cls = self.classifier(x)
        return cls

model = TestModel()
model.eval()

metric = RegionPerturbation(order="lerf")
metric_results = metric(model=model, x_batch=np.random.rand(2,2), y_batch=np.array([0,1]), a_batch=np.random.rand(2,2))
```

### Minimum acceptance criteria
- "lorf" -> "lerf" in https://github.com/understandable-machine-intelligence-lab/Quantus/blob/f94853b653700b8f499d203118664edf8ad8bef3/quantus/helpers/asserts.py#L109

**Comment by jang1suh** [comment]
Thank you very much!
